### PR TITLE
update local instructions

### PR DESCRIPTION
**Pull request by parthshahva**
Updated instructions and flag to allow for retention of bundles for testing purposes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,24 @@ This repository servers two purposes.
...
+Before you can run the tests, you need to setup your environment:
 
     pip install -r requirements.txt
-    CI_COMMIT_REF_NAME=staging make
+    # ping in #dcp-ops or #dcp-ops-help if you need valid service account credentials
+    export GOOGLE_APPLICATION_CREDENTIALS={PATH_TO_VALID_SERVICE_ACCOUNT_CREDS_WITH_ACCESS_TO_DSS}
+
+Instructions for running tests locally in python 3.6:
 
 Be aware that you will be uploading bundles to the staging deployment of the DCP and running secondary analysis.
```
Comment (xbrianh) [inline README.md]: Are these instructions specific to staging?

File: README.md [hunk 2]
```diff
@@ -8,13 +8,24 @@ This repository servers two purposes.
...
-The test currently takes about 18 minutes to run.
+The test currently takes about 120 minutes to run (longer for optimus, shorter for smart-seq-2).
+
+Also note that the test is currently setup to tombstone the bundles at the end of the run. You will need to set an env variable RETAIN_BUNDLES=True to disable this cleanup (ie if you need to retain the bundle for testing)
```
Comment (xbrianh) [inline README.md]: spelling nit: s/ie/i.e./

File: README.md (hunk 1 above)
Comment (parthshahva) [reply README.md]: made them more generic

File: README.md (hunk 2 above)
Comment (parthshahva) [reply README.md]: fixed

**Pull request merged**